File: deaf_grandma.py
def deaf_grandma():
    goodbye_counter = 0
    while True:
        question = input("Don't be shy, ask grandma a question: ")
        if question == "GOODBYE":
            if goodbye_counter == 1:
                print ("LATER, SKATER!")
                quit()
            else: 
                goodbye_counter += 1
                print("LEAVING SO SOON?")
        elif question == "":
            print("WHAT?!")
        elif question == question.upper():
            print("NO, NOT SINCE 1946!")
        else:
            print("SPEAK UP, KID!")
        continue

deaf_grandma()

File: deaf_grandma.js
function deafGrandma() {
    const prompt = require('prompt-sync')();
    let goodbyeCounter = 0;
    while (true) {
        const question = prompt('What do you want, kid? ');
        if (question === 'GOODBYE') {
            if (goodbyeCounter === 1) {
                console.log('LATER SKATER!');
                return;
            } else {
                goodbyeCounter += 1;
                console.log('LEAVING SO SOON?');
            };
        } else if (question === '') {
            console.log('WHAT?!');
        } else if (question === question.toUpperCase()) {
            console.log('NO, NOT SINCE 1946!');
        } else {
            console.log('SPEAK UP KID!');
        };
    };
};
deafGrandma();
